lightningd/opening/opening.c

```diff
@@ -209,7 +209,6 @@ static u8 *funder_channel(struct state *state,
 			  const struct utxo *utxos,
 			  const struct ext_key *bip32_base)
 {
-	const tal_t *tmpctx = tal_tmpctx(state);
```

Comment:
The opening daemon is so short-lived, and has no real loops, that we basically don't have to free memory.  When I first modified the code, I introduced a use-after-free bug, so I just removed tmpctx to match the fundee_channel code which was similarly naive.